Digital 1/Assets/Scripts/Bullet.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Digital1
```

Comment:
Get rid of namespaces. Project isn't big enough for them to be reasonably used.

Reply:
Gotcha, was trying their use for a bug fix and forgot to remove them.